qiskit/circuit/library/standard_gates/__init__.py

```diff
@@ -93,5 +92,6 @@
 from .x import MCXGate, MCXGrayCode, MCXRecursive, MCXVChain
 from .y import YGate, CYGate
 from .z import ZGate, CZGate
-
+# pylint: disable=cyclic-import
 from .multi_control_rotation_gates import mcrx, mcry, mcrz
+from ..generalized_gates.gms import MSGate
```

Comment:
Here, I think we have to keep the import from `ms` so that users who import from `standard_gates` will see the `DeprecationWarning` and be directed to `generalized_gates` (even though it will mean re-silencing the warning raised in `equivalence_library.py`.)

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/4979/commits/29bbc333d01f453f93a4970271eec7091e2fe404